packages/core/src/errors.ts

```diff
@@ -110,6 +111,34 @@ export class ConnectorNotFoundError extends Error {
   message = 'Connector not found'
 }
 
+export class ContractMethodNoResultError extends Error {
```

Comment:
Should we export this from the various `index.ts`s like the other errors?

Reply:
Yes!